tests/ModelTest.php

```diff
@@ -183,7 +183,14 @@ public function testSearchWithNoParams() {
    */
   public function testGetTypesWithoutParams() {
     $result = $this->model->getTypes();
-    $this->assertEquals(array('http://www.w3.org/2004/02/skos/core#Concept', 'http://www.w3.org/2004/02/skos/core#Collection', 'http://purl.org/iso25964/skos-thes#ConceptGroup', 'http://purl.org/iso25964/skos-thes#ThesaurusArray', 'http://www.skosmos.skos/test-meta/TestClass'), array_keys($result));
+    $this->assertEquals(
+        array(
+            'http://www.w3.org/2004/02/skos/core#Concept',
+            'http://www.w3.org/2004/02/skos/core#Collection',
+            'http://purl.org/iso25964/skos-thes#ThesaurusArray',
+            'http://purl.org/iso25964/skos-thes#ConceptGroup',
```

Comment:
The only change in this array is the order of `ThesaurusArray` and `ConceptGroup`. Not sure why these two swapped places in this array.